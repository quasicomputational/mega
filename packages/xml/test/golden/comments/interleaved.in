<foo>hello <!--A--> bye <!--B --> ok</foo>