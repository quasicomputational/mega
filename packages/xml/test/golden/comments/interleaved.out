<result><pre-root></pre-root><foo>hello <!--A--> bye <!--B --> ok</foo><post-root></post-root></result>